### ISSUE TITLE
Basic Instructions

### ISSUE THREAD

**Issue by boycad** [ShikyoKira/Project-New-Reign---Nemesis-Main#146]
Please add basic instructions, either to the page, or to the package.  I have missed them if they are already there.

ELI5: What am I looking at, and how do I turn it on?

...or is the hkxcmd.exe not supposed to start the tool?

**Comment by ShikyoKira** [comment]
After the beta test has been running for about a year, this is the first time i hear something asking for this.

Please go to this page to download the file: https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/releases

There should be a very obvious file located in nemesis_engine folder called "Nemesis Unlimited Behavior Engine.exe", you can't miss that and the basic instruction is in the read me of this main page

**Comment by boycad** [comment]
I'm not trying to be annoying.  It is quite likely that it has happened many time before, and the affected person just became dis-affected like I almost did.  

Its was quite easy to become lost.   Here is what I did: 1. download 2. extract 3. ...? (poke, poke) O, there's an .exe... 4. wait, what...?

Anyways, Id like to help.  Its a great project.  Those seeking more detailed instructions, such as the important step about using a mod manager or (less recommended) extract to /Data, should look at the Nexus page: https://www.nexusmods.com/skyrimspecialedition/mods/31667?tab=description

By all means, keep this issue closed.  I'll be contacting the author on via Nexus chat to see if he is interested in getting a more helpful Readme together.
